aws/delta-checkpoint/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "delta-checkpoint"
```

Comment:
i would add `lambda` as suffix or prefix to the name unless we plan to add other types of checkpoint daemon in the same package in the future.